api-reference/beta/api/connectorgroup_list_applications.md

```diff
@@ -1,6 +1,6 @@
 # List applications
 
-Retrieve a list of application objects associated with the connectorGroup.
+Retrieve a list of application objects associated with the connectorGroups.
```

Comment:
The original is correct.  Although I would phrase "a connectorGroup" rather than "the connectorGroup"